### PR TITLE
Fix doxygen comments

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: include/amici/solver_cvodes.h [hunk 1]
```diff
@@ -137,6 +148,11 @@ class CVodeSolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
+     * @param cv_mem
+     * @param y0
```
Comment (dweindl) [inline include/amici/solver_cvodes.h:154]: @FFroehlich : can you please fill in?

File: include/amici/solver_idas.h [hunk 2]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
     void setNonLinearSolverB(int which) const override;
 
   protected:
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
+     * @param t
+     * @param yout
+     * @param ypout
+     * @param tout
```
Comment (dweindl) [inline include/amici/solver_idas.h:114]: here too

File: include/amici/solver_idas.h [hunk 3]
```diff
@@ -135,6 +146,12 @@ class IDASolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
+     * @param ida_mem
+     * @param yy0
+     * @param yp0
```
Comment (dweindl) [inline include/amici/solver_idas.h:153]: here too

File: include/amici/solver_cvodes.h [hunk 4]
```diff
@@ -137,6 +148,11 @@ class CVodeSolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
+     * @param cv_mem
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:153]:
```suggestion
     * @param cv_mem pointer to CVODES solver memory object
```

File: include/amici/solver_cvodes.h [hunk 5]
```diff
@@ -137,6 +148,11 @@ class CVodeSolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:152]:
```suggestion
     * @brief resetState reset the CVODES solver to restart integration after a rhs discontinuity.
```

File: include/amici/solver_cvodes.h (hunk 1 above)
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:154]:
```suggestion
     * @param y0 new state vector
```

File: include/amici/solver_idas.h [hunk 6]
```diff
@@ -135,6 +146,12 @@ class IDASolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
+     * @param ida_mem
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:151]:
```suggestion
     * @param ida_mem pointer to IDAS solver memory object
```

File: include/amici/solver_idas.h [hunk 7]
```diff
@@ -135,6 +146,12 @@ class IDASolver : public Solver {
 
     void setSuppressAlg(bool flag) const override;
 
+    /**
+     * @brief resetState
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:150]:
```suggestion
     * @brief resetState reset the IDAS solver to restart integration after a rhs discontinuity.
```

File: include/amici/solver_idas.h [hunk 8]
```diff
@@ -135,6 +146,12 @@ class IDASolver : public Solver {
...
+     * @param ida_mem
+     * @param yy0
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:152]:
```suggestion
     * @param yy0 new state vector
```

File: include/amici/solver_idas.h (hunk 3 above)
Comment (FFroehlich) [inline include/amici/solver_idas.h:153]:
```suggestion
     * @param yp0 new state derivative vector
```

File: include/amici/solver_idas.h [hunk 9]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
     void setNonLinearSolverB(int which) const override;
 
   protected:
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:110]:
```suggestion
     * @param ida_mem pointer to IDAS solver memory object
```

File: include/amici/solver_idas.h [hunk 10]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
...
+     * @param t
+     * @param yout
+     * @param ypout
+     * @param tout
+     */
     void reInitPostProcess(void *ami_mem, realtype *t, AmiVector *yout,
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:116]:
```suggestion
    void reInitPostProcess(void *ida_mem, realtype *t, AmiVector *yout,
```

File: include/amici/solver_idas.h [hunk 11]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
     void setNonLinearSolverB(int which) const override;
 
   protected:
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
+     * @param t
+     * @param yout
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:112]:
```suggestion
     * @param yout new state vector
```

File: include/amici/solver_idas.h [hunk 12]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
     void setNonLinearSolverB(int which) const override;
 
   protected:
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
+     * @param t
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:111]:
```suggestion
     * @param t pointer to integration time
```

File: include/amici/solver_idas.h [hunk 13]
```diff
@@ -102,6 +105,14 @@ class IDASolver : public Solver {
...
+     * @param yout
+     * @param ypout
```
Comment (FFroehlich) [inline include/amici/solver_idas.h:113]:
```suggestion
     * @param ypout new state derivative vector
```

File: include/amici/solver_idas.h (hunk 2 above)
Comment (FFroehlich) [inline include/amici/solver_idas.h:114]:
```suggestion
     * @param tout anticipated next integration timepoint.
```

File: include/amici/solver_cvodes.h [hunk 14]
```diff
@@ -105,6 +109,13 @@ class CVodeSolver : public Solver {
 
     void reInitPostProcessB(realtype tnext) const override;
 
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
+     * @param t
+     * @param yout
+     * @param tout
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:117]:
```suggestion
     * @param tout  anticipated next integration timepoint.
```

File: include/amici/solver_cvodes.h [hunk 15]
```diff
@@ -105,6 +109,13 @@ class CVodeSolver : public Solver {
 
     void reInitPostProcessB(realtype tnext) const override;
 
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:114]:
```suggestion
     * @param cv_mem pointer to CVODES solver memory object
```

File: include/amici/solver_cvodes.h [hunk 16]
```diff
@@ -105,6 +109,13 @@ class CVodeSolver : public Solver {
...
+     * @param t
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:115]:
```suggestion
     * @param t pointer to integration time
```

File: include/amici/solver_cvodes.h [hunk 17]
```diff
@@ -105,6 +109,13 @@ class CVodeSolver : public Solver {
...
+     * @param yout
+     * @param tout
+     */
     void reInitPostProcess(void *ami_mem, realtype *t, AmiVector *yout,
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:119]:
```suggestion
    void reInitPostProcess(void *cv_mem, realtype *t, AmiVector *yout,
```

File: include/amici/solver_cvodes.h [hunk 18]
```diff
@@ -105,6 +109,13 @@ class CVodeSolver : public Solver {
 
     void reInitPostProcessB(realtype tnext) const override;
 
+    /**
+     * @brief Postprocessing of the solver memory after a discontinuity
+     * @param ami_mem
+     * @param t
+     * @param yout
```
Comment (FFroehlich) [inline include/amici/solver_cvodes.h:116]:
```suggestion
     * @param yout  new state vector
```

**Comment by dweindl** [comment]
Thx @FFroehlich

**Pull request merged**